Sources/MetriportSDK/MetriportClient.swift

```diff
@@ -278,31 +279,35 @@ extension SampleOrWorkout: Codable {
                                                    startDate: startDate,
                                                    endDate: endDate,
                                                    queryOption: queryOption) else {
-                metriportApi?.sendError(metriportUserId: metriportUserId, error: "Error unable to handle historical statistics")
+                metriportApi?.sendError(metriportUserId: metriportUserId, error: "Error unable to handle historical statistics type: \(type) start: \(startDate) end: \(endDate)")
```

Comment:
Can we have these dynamic data as additional information on the payload, instead of being part of the error message? By having those dynamic fields there, Sentry won't group them together, making it harder for us to manage them.

Maybe something like the `extra` fields we have on `capture.error()` and on Sentry SDK?